java/android/nnstreamer/src/main/jni/nnstreamer-android.h

```diff
@@ -25,6 +25,12 @@ extern "C" {
 extern jboolean
 nnstreamer_native_initialize (JNIEnv *env, jobject context);
 
+/**
+ * @brief Get the data path of an application, extracted using getFilesDir().
+ */
+extern const char *
```

Comment:
Add a doxygen note that the caller does not need to free the returned pointer.

Reply:
Updated, plz review changes again.